### DISCUSSION TITLE
GitHub actions dont start anymore

### DISCUSSION THREAD

**Discussion by mStirner** [community/community#26783] [Actions]
<p>When i started to use GitHub actions they worked fine.<br>
But now, they dont start any more.</p>
<p>Repo location: <a href="https://github.com/OpenHausIO/backend" class="inline-onebox" rel="noopener nofollow ugc">GitHub - OpenHausIO/backend</a><br>
Even my private workflows forks dont work/run.</p>
<p>Error message: <code>This request was automatically failed because there were no enabled runners online to process the request for more than 1 days.</code></p>

**Comment by cschleiden** [comment, marked as answer]
<p>Hello,</p>
<p>specifying <code>runs-on: [ubuntu-18.04, ubuntu-latest]</code> in the workflow means that the job is looking for a runner that satisfies <strong>both</strong> labels. Since <code>ubuntu-latest</code> points to <code>20.x</code> at the moment, we do not have any hosted runners which satisfy both labels.</p>
<p>If you want to run the workflow job on an <code>ubuntu-18.04</code> <strong>and</strong> <code>ubuntu-latest</code> runners, you’d have to include that in your matrix configuration: see here <a href="https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-running-with-multiple-operating-systems" rel="noopener nofollow ugc">Workflow syntax for GitHub Actions - GitHub Docs</a> for an example.</p>
<p>Otherwise chose <em>either</em> <code>ubuntu-18.04</code> or <code>ubuntu-latest</code>.</p>

**Comment by mStirner** [comment]
<p>Oh god, thank you very much.<br>
I feel so dumb right now…  -_-</p>
